udata/core/dataset/models.py

```diff
@@ -150,6 +152,8 @@ def is_available(self):
 
 
 class Resource(ResourceMixin, WithMetrics, db.EmbeddedDocument):
+    ''' Local file, remote file or API provided by the original provider of the
```

Comment:
👍 for the docstring but we use this convention:
- No leading space for one liners
- Begins and stops with a line break for multiple lines